src/test/ui/consts/const-fn-not-safe-for-const.rs

```diff
@@ -18,12 +18,10 @@ static Y: u32 = 0;
 
 const fn get_Y() -> u32 {
     Y
-        //~^ ERROR E0013
```

Comment:
With the `const_fn` feature gate we do not catch this anymore. Stable const fns will still trigger this as shown in https://github.com/rust-lang/rust/blob/56237d75b4271a8a2e0f47d86ea76ebf6d966152/src/test/ui/consts/min_const_fn/min_const_fn.rs#L90